crates/metadata/src/specs.rs

```diff
@@ -138,6 +138,7 @@ impl ContractSpecBuilder<Invalid> {
 
 impl<S> ContractSpecBuilder<S> {
     /// Sets the messages of the contract specification.
+    #[must_use]
```

Comment:
If this is `return_self_not_must_use` then I the docs say we can just decorate the `ContractSpecBuilder` type itself with that rather than all methods